docs/en/core/defaults-files.md

```diff
@@ -81,17 +81,60 @@ variables:
 You can also make use of this templating system yourself. If you want to write a custom template, you can add statements such as `$if(myvariable)$$myvariable$$endif$` and insert the variable `myvariable` into any frontmatter or defaults file, and it will be replaced with whatever value you set it to.
 
 
-### Microsoft Word Template (.docx)
+### Reference Documents (.docx, .odt, .pptx) 
 
-For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+For export as Microsoft Word document, OpenDocument, or PowerPoint presentation, Pandoc uses a reference document that serves as a style template. This section is written in relation to Word docx, but the same should apply for either ODT or PPTX. 
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions. 
+
+##### Option 1: Using Pandoc from Command line or Terminal 
+
+In the command line: `pandoc -o "custom-reference.docx" --print-default-data-file reference.docx`
+
+To adjust the name or location of where the document is saved, you can change `"custom-reference.docx"` to include the absolute path: `"/Users/[user]/Downloads/project-reference.docx"`
+
+##### Option 2: Download from Pandoc Github repository
+
+ In the Pandoc repo the documents are unzipped, which makes downloading them somewhat complicated. 
+
+1. Open https://github.com/jgm/pandoc/
+2. In the green Code button, select "Download zip"
+3. Open the folder, find /pandoc-master/data/docx, and ZIP (compress) the docx folder. 
+4. Change ".zip" to ".docx" in the file name
+5. Open with Word
+
+This method can lead to errors, and it is recommended to follow Option 1. 
+
+##### Option 3: Finding documents produced by others
+
+Formatted Pandoc reference documents can be found on Github or elsewhere. Use these at your own risk. 
+
+Examples: 
+- https://github.com/hokorobi/pandoc-docx-reference/blob/master/reference.docx
```

Comment:
Remove these links as we cannot guarantee for their safety or that they won't turn harmful in the future. Feel free to add helpful google search terms instead so that users can search for themselves.